utils/src/ooinstall/cli_installer.py

```diff
@@ -339,7 +339,10 @@ def get_hosts_to_run_on(oo_cfg, callback_facts, unattended, force):
                 # new nodes.
             elif host.node:
                 click.echo("{} is already an OpenShift Node".format(host))
-                hosts_to_run_on.remove(host)
+                # force is only used for reinstalls so we don't want to remove
+                # anything.
+                if not force:
+                    hosts_to_run_on.remove(host)
```

Comment:
@smunilla, without this the following installer config wasn't working with `--force`:

~~~
ansible_log_path: /tmp/ansible.log
ansible_ssh_user: root
hosts:
- hostname: ose3-master.example.com
  ip: 192.168.133.2
  master: true
  node: true
  public_hostname: ose3-master.example.com
  public_ip: 192.168.133.2
- hostname: ose3-node1.example.com
  ip: 192.168.133.3
  node: true
  public_hostname: ose3-node1.example.com
  public_ip: 192.168.133.3
variant: openshift-enterprise
variant_version: '3.1'
version: v1
~~~